segmentation/include/pcl/segmentation/euclidean_cluster_comparator.h

```diff
@@ -50,31 +50,25 @@ namespace pcl
     *
     * \author Alex Trevor
     */
-  template<typename PointT, typename PointNT, typename PointLT>
+  template<typename PointT, typename PointLT>
```

Comment:
If labels are now optional, can the template label default to PointLT = pcl::Label?